src/Modal/Modal.module.scss

```diff
@@ -8,15 +8,24 @@
   bottom: 0;
   left: 0;
   background: rgba(0, 0, 0, 0.8);
+  animation: fadeIn 0.5s cubic-bezier(0.165, 0.84, 0.44, 1) forwards;
 }
 
 .dialog {
   position: fixed;
   z-index: 1001;
-  max-width: 90%;
   top: 50%;
   left: 50%;
   transform: translate3d(-50%, -50%, 0);
+  max-width: 100%;
+  max-height: 100%;
```

Comment:
From the UI/UX point of view, it's much better to leave empty (dimmed) space between the window and modal.
```scss
   max-width: 90%;
   max-height: 90%;
```